cobc/cobc.c

```diff
@@ -9270,6 +9301,14 @@ main (int argc, char **argv)
 	/* Process command line arguments */
 	iargs = process_command_line (argc, argv);
 
+/* initialize the iconv struct after reading the command line*/
+#ifdef HAVE_ICONV_H
+	cb_iconv.alphanumeric = iconv_open("ISO-8859-15", cb_iconv.source);
+	cb_iconv.national = iconv_open("UTF-16LE", cb_iconv.source);
+	cb_iconv.utf8 = iconv_open("UTF-8", cb_iconv.source);
+#endif
+
```

Comment:
I think this should go after "other one-time initialization" down a bunch of lines below, but even better - be only initialized (at least national + utf8) if one national/utf8 literal / data is seen - as na huge amount of programs won't ever use that.